k8s/jobs/mediawikiUpdate.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: mediawiki-update-
+  namespace: adhoc-jobs
+spec:
+  ttlSecondsAfterFinished: 86400
+  backoffLimit: 0
+  template:
+    metadata:
+      name: mediawiki-update
+    spec:
+      containers:
+        - name: mediawiki-update
+          command:
+            - bash
+            - -c
+            - |
+                set -euo pipefail
+
+                log_info() {
+                  printf 'INFO: %s\n' "$*"
+                }
+
+                log_error() {
+                  printf 'ERROR: %s\n' "$*" >&2
+                }
+
+                log_info "Setting ${WBS_DOMAIN} to readOnly"
+                curl --fail --location --silent --request PUT \
+                "http://${PLATFORM_API_BACKEND_HOST}/backend/setWikiReadOnly?domain=${WBS_DOMAIN}&readOnly=true"
+
+                export MW_INSTALL_PATH=/var/www/html/w/
+
+                log_info "Running MediaWiki update.php for ${WBS_DOMAIN}"
+                php /var/www/html/w/maintenance/run.php update --quick
```

Comment:
for tom: right now if this errors its going to immediately exit and there's not going to be this nice error logging based on status. should we just turn off `set -e` for a moment?